AWSClientRuntime/Sources/Auth/AWSCredentialsProviderProfileOptions.swift

```diff
@@ -17,44 +17,15 @@ public struct AWSCredentialsProviderProfileOptions {
     
     public let credentialsFileNameOverride: String?
     
-    public init(configFileNameOverride: String? = nil,
-                profileFileNameOverride: String? = nil,
-                credentialsFileNameOverride: String? = nil,
-                shutdownCallback: ShutDownCallback? = nil) {
+    public init(
+        configFileNameOverride: String? = nil,
+        profileFileNameOverride: String? = nil,
+        credentialsFileNameOverride: String? = nil,
+        shutdownCallback: ShutDownCallback? = nil
+    ) {
         self.configFileNameOverride = configFileNameOverride
         self.profileFileNameOverride = profileFileNameOverride
         self.credentialsFileNameOverride = credentialsFileNameOverride
         self.shutdownCallback = shutdownCallback
     }
 }
-
-extension AWSCredentialsProviderProfileOptions {
-    func toCRTType() -> CredentialsProviderProfileOptions {
-        return CredentialsProviderProfileOptions(configFileNameOverride: configFileNameOverride,
-                                                 profileFileNameOverride: profileFileNameOverride,
-                                                 credentialsFileNameOverride: credentialsFileNameOverride,
-                                                 shutdownOptions: shutdownCallback)
-    }
-}
-
-struct CredentialsProviderProfileOptions: CRTCredentialsProviderProfileOptions {
```

Comment:
not required?

Reply:
Nope. This struct no longer has any purpose. 
Checkout the related factory method in [AwsCredentialsProvider]](https://github.com/awslabs/aws-sdk-swift/pull/777/files#diff-c858c06a42a458b91bb72c1ce5190f515a40cf4463775e74d8f8f91d4e53c506) to see why its no needed.